lib/modules/showImages.js

```diff
@@ -968,6 +974,18 @@ async function checkElementForMedia(element) {
 
 	if (thing) thingExpandoBuildListeners.fire(thing);
 
+	if (module.options.hideOtherExpandos.value) {
```

Comment:
@larsjohnsen is this a good place to add this listener? And do you think it might interfere with anything else that automatically opens/closes expandos?